StringUtil.java

```diff
@@ -6,21 +6,66 @@
  * Input:
  */
 public class StringUtil {
-
+    
     private static String[] URLParser(String url){
-        //WRITE YOUR CODE HERE..
-        //parse the url and get all the get parameters and return an array of Strings
-        //Input: google.com/api?param1=value1&param2=value2
-        //Output: array of strings [] = { {param1:value1}, {param2:value2} }
+       String result;
+       String[] tempArray = new String[2];
+       String[] splittedUrl = url.split("[\\?\\&]");
+       String[] resultArray = new String[5];
```

Comment:
Do not hard code the length of the array. If the splittedUrl array size is 7, then you cannot store the values in resultArray.

``String[] resultArray = new String[splittedUrl.length];``